python-test-samples/apigw-sqs-lambda-sqs/README.md

```diff
@@ -0,0 +1,153 @@
+   
+[Top](#contents)
+
+---
+
+## Sample project description
+
+The sample project allows a user to call an API endpoint (using /inbox) and generate a custom "test/hello" message. The user can also track the result of the processing of it in the cloud (using /outbox). The following diagram demonstatred the architecture and flow. The following diagram has been created using the [AWS Application Composer](https://aws.amazon.com/application-composer/), which can help you visually design and build serverless applications quickly.
+
+![Event Sequence](img/architecture.png)
+
+This project consists of an [API Gateway](https://aws.amazon.com/api-gateway/), two [AWS Lambda](https://aws.amazon.com/lambda) functions, and two [Amazon SQS](https://aws.amazon.com/sqs) standard queues which are using 2 DLQ queues accordinally for error handling.
+
+The Sequence is (corresponding steps 1-7  on the diagram): 
+
+1. User is using the test client to invoke an API call (POST /inbox) to send a message/job/test to be later on processed in the backend by the ProcessInputQueue lambda function.
+2. API GW is sending the message payload into the InputQueue.
+3. InputQueue is triggering the ProcessInputQueue lambda function to process the message/job/test in the queue. this is where the lambda can be enhanced to do further processing/testing as needed by you. It's up to the user of this sample to decide what this lambda shuold eventualy do, as it can be extended and adapted to the testing needs. 
+4. When the ProcessInputQueue lambda finished its processing, it sends the testing result to the OutputQueue (if needed - please adapt the result JSON/Message). The result will be kept in the queue until consumed by the User.
+5. User is using the test client to invoke an API call (GET /outbox) to receive the testing result/message of the test it issued on step 1.
+6. API GW is triggering the CheckOutputQueue lambda function to recieve the message from the OutputQueue.
+7. CheckOutputQueue lambda function is receiving the the message from the OutputQueue and delivers it back to the test client via the API GW.
+
+
+
+---
+
+## Prerequisites
+The SAM CLI extends the AWS CLI that adds functionality for building and testing serverless applications. It contains features for building your application locally, deploying it to AWS, and emulating AWS services locally to support automated unit tests.  
+
+To use the SAM CLI, you need the following tools.
+
+* SAM CLI - [Install the SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
+* Python 3.9 - [Install Python 3.9](https://www.python.org/downloads/)
+
+    In case you are using AWS cloud9, currently the default Python version is 3.7. you can check [here](https://repost.aws/questions/QU14iutbqtSsm1gHwQwt02pA/upgrade-to-python-3-9-on-cloud-9) how to update it.
+
+
+```
```

Comment:
add the word "shell" after the three open quotes for style

Reply:
not sure I understand ... when I added it it didn't do any style change

Reply:
The 3 quotes on line 68 should be moved to line 72.

Add ```shell

Reply:
done